if any cxxflags are set it skips all the checks
--
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17020e1..3a86366 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,7 +97,7 @@ endif()
 
 # check for SSE flags
 include("${PCL_SOURCE_DIR}/cmake/pcl_find_sse.cmake")
-if(PCL_ENABLE_SSE AND "${CMAKE_CXX_FLAGS}" STREQUAL "${CMAKE_CXX_FLAGS_DEFAULT}")
+if(PCL_ENABLE_SSE)
   PCL_CHECK_FOR_SSE()
 endif()
 
@@ -109,14 +109,12 @@ endif()
 
 # ---[ Unix/Darwin/Windows specific flags
 if(CMAKE_COMPILER_IS_GNUCXX)
-  if("${CMAKE_CXX_FLAGS}" STREQUAL "${CMAKE_CXX_FLAGS_DEFAULT}")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 7)
       string(APPEND CMAKE_CXX_FLAGS " -Wabi=11")
     else()
       string(APPEND CMAKE_CXX_FLAGS " -Wabi")
     endif()
     string(APPEND CMAKE_CXX_FLAGS " -Wall -Wextra -Wno-unknown-pragmas -fno-strict-aliasing -Wno-format-extra-args -Wno-sign-compare -Wno-invalid-offsetof -Wno-conversion ${SSE_FLAGS} ${AVX_FLAGS}")
-  endif()
 
   if(PCL_WARNINGS_ARE_ERRORS)
     string(APPEND CMAKE_CXX_FLAGS " -Werror -fno-strict-aliasing")
